src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -81,22 +96,51 @@ export class CalciteValueListItem {
 
   getPickListRef = (el) => (this.pickListItem = el as HTMLCalcitePickListItemElement);
 
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.key === " ") {
+      this.handleActivated = !this.handleActivated;
+    }
+  };
+
+  handleBlur = () => {
+    this.handleActivated = false;
+  };
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
+  renderHandle() {
+    const { icon } = this;
+    if (icon === ICON_TYPES.grip) {
+      return (
+        <a
+          class={{ [CSS.handle]: true, [CSS.handleActivated]: this.handleActivated }}
+          tabindex="0"
+          data-js-handle="true"
+          data-activated={this.handleActivated}
+          onKeyDown={this.handleKeyDown}
+          onBlur={this.handleBlur}
+        >
+          <CalciteIcon size="16" path={drag16} />
+        </a>
+      );
+    }
+  }
+
   render() {
     return (
-      <Host>
+      <Host data-id={this.guid}>
+        {this.renderHandle()}
         <calcite-pick-list-item
           compact={this.compact}
           ref={this.getPickListRef}
           disabled={this.disabled}
           selected={this.selected}
           metadata={this.metadata}
-          icon={this.icon}
+          icon={null}
```

Comment:
question: Do you need to pass null here?

Reply:
doubled-checked. The default is null. Removing.